routes/index.js

```diff
@@ -51,11 +51,22 @@ router.get('/', async (req, res) => {
     assigmentQ.add(async () => {
       var username = req.session.username
 
+      if (username) {
+        const valid = await users.isUserAssignmentValid(username)
```

Comment:
This is basically a freshness check that would allow the blocklist to be updated during a lab. If an attendee was assigned a user that's now blocked this will remove that assignment from their session